devices.js

```diff
@@ -13384,6 +13384,23 @@ const devices = [
         description: 'LED white',
         extend: preset.light_onoff_brightness(),
     },
+    {
+        fingerprint: [
+            {type: 'Router', manufacturerName: 'AwoX', modelID: 'TLSR82xx', endpoints: [
+                {ID: 1, profileID: 260, deviceID: 258, inputClusters: [0, 3, 4, 5, 6, 8, 768, 4096], outputClusters: [6, 25]},
+                {ID: 3, profileID: 49152, deviceID: 258, inputClusters: [65360, 65361], outputClusters: [65360, 65361]},
+            ]},
+        ],
+        model: '33943',
+        vendor: 'AwoX',
+        description: 'LED RGB & brightness',
+        fromZigbee: [fz.on_off, fz.brightness, fz.level_config, fz.ignore_basic_report],
```

Comment:
The `fromZigbee`, `toZigbee` and `exposes` lines can be replaced with: `extend: preset.light_onoff_brightness_colortemp_color(),`

Reply:
ok added :)